diff --git a/node_modules/@keystone-next/fields/dist/fields.cjs.dev.js b/node_modules/@keystone-next/fields/dist/fields.cjs.dev.js
index 0aac580..37df075 100644
--- a/node_modules/@keystone-next/fields/dist/fields.cjs.dev.js
+++ b/node_modules/@keystone-next/fields/dist/fields.cjs.dev.js
@@ -288,7 +288,7 @@ const decimal = (_ref = {}) => {
       kind: 'scalar',
       mode: 'optional',
       scalar: 'Decimal',
-      nativeType: `Decimal(${precision}, ${scale})`,
+      nativeType: `Decimal()`,
       index
     })(_objectSpread(_objectSpread({}, config), {}, {
       input: {
diff --git a/node_modules/@keystone-next/fields/dist/fields.cjs.prod.js b/node_modules/@keystone-next/fields/dist/fields.cjs.prod.js
index 0aac580..37df075 100644
--- a/node_modules/@keystone-next/fields/dist/fields.cjs.prod.js
+++ b/node_modules/@keystone-next/fields/dist/fields.cjs.prod.js
@@ -288,7 +288,7 @@ const decimal = (_ref = {}) => {
       kind: 'scalar',
       mode: 'optional',
       scalar: 'Decimal',
-      nativeType: `Decimal(${precision}, ${scale})`,
+      nativeType: `Decimal()`,
       index
     })(_objectSpread(_objectSpread({}, config), {}, {
       input: {
diff --git a/node_modules/@keystone-next/fields/dist/fields.esm.js b/node_modules/@keystone-next/fields/dist/fields.esm.js
index 7338df4..6c2166c 100644
--- a/node_modules/@keystone-next/fields/dist/fields.esm.js
+++ b/node_modules/@keystone-next/fields/dist/fields.esm.js
@@ -277,7 +277,7 @@ const decimal = (_ref = {}) => {
       kind: 'scalar',
       mode: 'optional',
       scalar: 'Decimal',
-      nativeType: `Decimal(${precision}, ${scale})`,
+      nativeType: `Decimal()`,
       index
     })(_objectSpread(_objectSpread({}, config), {}, {
       input: {
